developer-workflow/c-api.rst

```diff
@@ -19,8 +19,13 @@ The C API is divided into these tiers:
 Each tier has different stability and maintenance requirements to consider
 when you add or change definitions in it.
 
-The compatibility guarantees for public C API are explained in the
-user documentation, ``Doc/c-api/stable.rst`` (:ref:`python:stable`).
+The public backwards compatibility guarantees for public C API are explained
+in the user documentation, ``Doc/c-api/stable.rst`` (:ref:`python:stable`).
+C language compatibility guarantees are in ``Doc/c-api/intro.rst``
+(:ref:`python:api-intro`).
+
+Internally, we need to be more careful about compatibility than what we promise
+publicly. See :ref:`public-capi` for details.
```

Comment:
I don't understand this paragraph. What do you mean by "internally"? The internal C API?

Reply:
Perhaps "Within the core development group" is a better phrase than "internally"?